pointpats/quadrat_statistics.py

```diff
@@ -210,9 +211,9 @@ def __init__(self, pp, lh):
         self.count_row_odd = int(math.ceil(range_y/(self.semi_height*2)))
 
         # quadrat number
-        self.num = self.count_row_odd * (self.count_column/2 +
+        self.num = self.count_row_odd * (int(self.count_column/2) +
```

Comment:
It's good style (I feel) to use the explicit floor division operator `a//b` instead of `int(a/b)`, since it's a little more explicit. No pressure to revise this, but it looks clearer to use `//` and `%` in tandem when doing work on grids.